tests/test_interconnect/test_interconnect_cgra.py

```diff
@@ -86,6 +86,7 @@ def test_interconnect_point_wise(batch_size: int, cw_files, add_pd, io_sides):
         tester.expect(circuit.interface[dst_name], num_1 * num_2)
 
     with tempfile.TemporaryDirectory() as tempdir:
+        tempdir = "temp"
```

Comment:
why this?

Reply:
Oops that was for debugging purpose. I will remove it.